viewer/packages/sector-parser/tests/GltfSectorParser.test.ts

```diff
@@ -5,7 +5,8 @@
 import fs from 'fs';
 
 import 'jest-extended';
-import GltfSectorParser, { RevealGeometryCollectionType } from '../src/GltfSectorParser';
+import { GltfSectorParser } from '../src/GltfSectorParser';
+import { RevealGeometryCollectionType } from '../src/types';
 
 describe('GltfSectorParser Tests', () => {
```

Comment:
```suggestion
describe('GltfSectorParser', () => {
```
I think we should keep the naming scheme we have or switch to a new for all tests and not use several. Also, I've been thinking of if we should do this instead:
```
describe(GltfSectorParser.name, () => {
```
Which would avoid names getting out of sync.